flickit-assessment-kit/src/main/java/org/flickit/assessment/kit/common/ErrorMessageKey.java

```diff
@@ -70,6 +70,10 @@ public class ErrorMessageKey {
 
     public static final String GET_KIT_EDITABLE_INFO_KIT_ID_NOT_NULL = "get-kit-editable-info.kitId.notNull";
 
+    public static final String GET_SUBJECT_DETAIL_KIT_ID_NOT_NULL = "get-subject-detail.kitId.notNull";
```

Comment:
replace `GET_SUBJECT_DETAIL` with `GET_KIT_SUBJECT_DETAIL` in this and the following messages.